docs/source/conf.py

```diff
@@ -212,7 +214,7 @@
 # Note the underscore SHOULD be used below as this is how the static folder is
 # named by sphinx on generation.
 html_context = {
-    "css_files": ["_static/custom.css"],
+    "css_files": ["_static/custom.css", "_static/pygments.css", "_static/css/theme.css"],
```

Comment:
Can you revert this line as well? Can drop pygments css changes I think, but we need the `_static/css/theme.css` listed